applications/serial_lte_modem/prj.conf

```diff
@@ -62,9 +62,9 @@ CONFIG_HTTP_PARSER_URL=y
 CONFIG_FOTA_DOWNLOAD=y
 CONFIG_FOTA_DOWNLOAD_PROGRESS_EVT=y
 CONFIG_DFU_TARGET=y
-CONFIG_DOWNLOAD_CLIENT=y
-CONFIG_DOWNLOAD_CLIENT_STACK_SIZE=4096
-CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE=2048
+CONFIG_DOWNLOADER=y
```

Comment:
We are seeing random failures with application fota update, where the fota package is downloaded successfully, but after the reset, the new package is not updated and mcuboot_swap_type() is BOOT_SWAP_TYPE_NONE.

The issue is random, but can only take place if RTT-traces are activated, when the reset is performed (with AT#XRESET). 

Only takes place with 9151 (and likely 9161), but not with 9160.

Reply:
It takes place with nRF9161 as well.

Reply:
We verified that the image is uploaded to the secondary slot correctly, so this issue is not directly related to downloader. Rather, timings are changed a bit, which likely causes the issue. After the image is uploaded, the device apparently resets, which causes bootloader to revert the upgrade and load the old application image instead.